drv/fe310-rtc/src/main.rs

```diff
@@ -5,19 +5,27 @@
 #![no_std]
```

Comment:
Did you check if there is any different in interrupt latency as seen by the task with and without this change?

Reply:
There will definitely be an increase in interrupt latency, although we don't have any simple way to test it at the moment. There is an additional task switch that is needed as the driver will have to notify the task, and then another set of context switches so that the notified task gets control.

Reply:
Right. I think we should evaluate what that additional latency is. For some of our use cases, we might not be able to tolerate long latency. Can you please raise an issue to track this activity?

Reply:
Agreed, and will do.